src/js/modules/infragistics.ui.editors.js

```diff
@@ -38,36 +38,168 @@
+				```
+				//Initialize
+				$(".selector").%%WidgetName%%({
+					allowNullValue : false
+				});
+
+				//Get
+				var allowNullValue = $(".selector").%%WidgetName%%("option", "allowNullValue");
+
+				//Set
+				$(".selector").%%WidgetName%%("option", "allowNullValue", false);
+				```
 			*/
 			allowNullValue: false,
-			/* type="string|number|null" Gets/Sets the representation of null value. In case of default the value for the input is set to null, which makes the input to hold an empty string */
+			/* type="string|number|null" Gets/Sets the representation of null value. In case of default the value for the input is set to null, which makes the input to hold an empty string
+				```
+				//Initialize
+				$(".selector").%%WidgetName%%({
+					nullValue : null
+				});
+
+				//Get
+				var nullValue = $(".selector").%%WidgetName%%("option", "nullValue");
+
+				//Set
+				$(".selector").%%WidgetName%%("option", "nullValue", null);
+				```
+			*/
 			nullValue: null,
-			/* type="string" Sets the name attribute of the value input. This input is used to sent the value to the server. In case the target element is input and it has name attribute, but the developer has set the inputName option, so this option overwrites the value input and removes the attribute from the element. */
+			/* type="string" Sets the name attribute of the value input. This input is used to sent the value to the server. In case the target element is input and it has name attribute, but the developer has set the inputName option, so this option overwrites the value input and removes the attribute from the element.
+			```
+			//Initialize
+				$(".selector").igPercentEditor({
+				  inputName : "textField"
+				});
+
+				//Get
+				var inputName = $(".selector").igPercentEditor("option", "inputName");
+
+			//Set
+			$(".selector").igPercentEditor("option", "inputName", "textField");
+			```
+			*/
 			inputName: null,
-			/* type="bool" Gets/Sets the readonly attribute. Does not allow editing. Disables all the buttons and iteracitons applied. On submit the current value is sent into the request.*/
+			/* type="bool" Gets/Sets the readonly attribute. Does not allow editing. Disables all the buttons and iteracitons applied. On submit the current value is sent into the request.
+			```
+				//Initialize
+				$(".selector").%%WidgetName%%({
+					readOnly : true
+				});
+
+				//Get
+				var readOnly = $(".selector").%%WidgetName%%("option", "readOnly");
+
+				//Set
+				$(".selector").%%WidgetName%%("option", "readOnly", true);
+			```
+			*/
 			readOnly: false,
-			/* type="bool" Gets/Sets the disabled attribute.Does not allow editing. Disables all the buttons and iteracitons applied. On submit the current value is not sent into the request*/
+			/* type="bool" Gets/Sets the disabled attribute.Does not allow editing. Disables all the buttons and iteracitons applied. On submit the current value is not sent into the request.
```

Comment:
"Gets/Sets the disabled attribute for the input. If set to true the input is disabled, and all buttons and interactions are disabled. On submit the value is not submitted."